src/sentry/workflow_engine/nowa/logic.py

```diff
@@ -0,0 +1,114 @@
+import uuid
```

Comment:
just have functions scattered around here, will find a home for them, not sure where right now